api/workOrder/woModel.js

```diff
@@ -16,9 +16,24 @@ const create = async (wo) => {
   return db('workOrders').insert(wo).returning('*');
 };
 
+const update = (id, wo) => {
+  console.log(wo);
+  return db('workOrders')
+    .where({ id: id })
```

Comment:
here and on line 12 the id column in the database is named "requestId" so it should be ```.where({requestId: id})

Reply:
you're right. fixed, thank you

Reply:
probably just ID would be simpler. Yeah, I don't think we need both. I'm running into more linting problems I will switch it back to being ID, run prettier again and then hopefully we can get this to pass all the build tests again

Reply:
I'm going to go ahead and change the schema to use ID then and get that pull-request in